integration-tests/luks/create-with-integrity.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+
+# requirements: disk /dev/sdc with empty partition sdc1
+
+
+from storage import *
+from storageitu import *
+
+
+set_logger(get_logfile_logger())
+
+environment = Environment(False)
+
+storage = Storage(environment)
+storage.probe()
+
+staging = storage.get_staging()
+
+sdc1 = Partition.find_by_name(staging, "/dev/sdc1")
+
+luks = to_luks(sdc1.create_encryption("cr-test"))
```

Comment:
The create_encryption function without providing the encryption type is deprecated.

Reply:
Done.